cas/cas.h

```diff
@@ -36,7 +36,7 @@
 #include "rpc/at.h"             /* m0_rpc_at_buf */
 #include "rpc/at_xc.h"          /* m0_rpc_at_buf_xc */
 #include "fop/fom_generic.h"    /* m0_fop_mod_rep */
```

Comment:
"fop/fom_generic.h" can be removed here.
The `m0_fop_mod_rep` is not defined there any longer.

Reply:
I see it is fixed.